packages/sdk/.example.env

```diff
@@ -0,0 +1 @@
+VITE_ANVIL_ALCHEMY_KEY="<key>"
```

Comment:
Seems like this only works with Alchemy - is that an implementation choice on our side or is it inherited from Anvil?

Reply:
No this is just an implementation choice to keep the env file to one variable since we can share a single key between chains on alchemy